tests/foreman/api/test_contentview.py

```diff
@@ -165,14 +160,13 @@ def test_cv_associate_puppet_repo_negative(self):
             ).create()
 
     @run_only_on('sat')
-    def test_cv_associate_composite_dupe_repos_negative(self):
+    def test_negative_associate_composite_dupe_content(self):
```

Comment:
this does not look like a composite test? may be remove `composite` from the name?